tsdb/db_test.go

```diff
@@ -1030,7 +1030,7 @@ func TestTombstoneClean(t *testing.T) {
 	for _, c := range cases {
 		// Delete the ranges.
 
-		// create snapshot
+		// create snapshot.
```

Comment:
While you are on it, we can capitalize the first letter :) (also the other comment)